vcd/resource_vcd_org.go

```diff
@@ -148,8 +159,8 @@ func resourceOrgDelete(d *schema.ResourceData, m interface{}) error {
 
 	//fetches org
 	log.Printf("Reading Org with id %s", d.State().ID)
-	org, err := govcd.GetAdminOrgByName(vcdClient.VCDClient, d.Get("name").(string))
-	if err != nil || org == (govcd.AdminOrg{}) {
+	org, err := vcdClient.VCDClient.GetAdminOrgByName(d.Get("name").(string))
```

Comment:
I think we technical debt issue to cover this `vcdClient.VCDClient` ? what do you think? :)

Reply:
Please explain.

Reply:
Does this vcdClient.VCDClient look nice for you? :)

Reply:
It doesn't, but then there are many ugly things in this repository.

What do you suggest?
And should it be done in this PR?

Reply:
No don't do in this PR. Maybe we should or have Issue with such notes regarding what to fix?